targets/TARGET_RDA/TARGET_UNO_91H/lib/README.md

```diff
@@ -0,0 +1,7 @@
+This directory tree contains binaries build from RDA SDK modified for Mbed OS and released under Permissive Binary License.
+
+libhal files in the subfolders are generated with toolchains:
+
+Arm Compiler 5 - version 5.06u1
+GNU Arm Embedded - version 4.9.3
```

Comment:
One thing - this should be updated to use 6, can you update?

Reply:
updated